rust/nasl-cli/README.md

```diff
@@ -202,6 +202,41 @@ Options:
   -h, --help   Print help
 ```
 
+### scan-config
+
+Transforms a scan-config from gvmds data-objects to scan json of [openvasd](https://greenbone.github.io/scanner-api/#/scan/create_scanl).
+
+To set the target and credentials you can pipe a partial scan json into `nasl-cli scan-config` by providing `-i` flag.
+
+As an example we assume that the data-objects feed is in `~/src/greenbone/data-objects/content/22.04` while the vulnerability feed is in `~/src/greenbone/vulnerability-tests/nasl/common` and we want to create a scan to verify localhost with a discovery and full and fast policy on the openvas default portlist.
+
+For that we need to execute:
+
+```
+echo '{ "target": { "hosts": ["localhost"], "ports": [] }, "vts": [] }'| \
+nasl-cli scan-config -i -p ~/src/greenbone/vulnerability-tests/nasl/common -l ~/src/greenbone/data-objects/content/22.04/port-lists/openvas-default-c7e03b6c-3bbe-11e1-a057-406186ea4fc5.xml ~/src/greenbone/data-objects/content/22.04/scan-configs/discovery-8715c877-47a0-438d-98a3-27c7a6ab2196.xml | \
+nasl-cli scan-config -i -p ~/src/greenbone/vulnerability-tests/nasl/common  ~/src/greenbone/data-objects/content/22.04/scan-configs/full-and-fast-daba56c8-73ec-11df-a475-002264764cea.xml
+```
+
+Be aware to resolve the families within a scan-config nasl-cli requires to read the complete feed. Depending on your system that may require some time on each scan-config run.
```

Comment:
mhm, it parses the complete feed in any case before checking the scan-config. But I see it can be confusing. How about:

``` 
Be aware that each call does a description run of a defined feed to gather the meta data, depending on your system and the size of the feed it requires some time.
```